Header header
float64 mh
float64 mh_dot
